### PR TITLE
Scoring Pallet

### PR DESCRIPTION
**Pull request by rohibhan**

### CODE REVIEW

File: pallets/score/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+		ScoreOf,
+	>;
+	pub type JournalInputOf<T> =
+		JournalInput<JournalDetailsOf<T>, EntryHashOf<T>, EntitySignatureOf<T>>;
+	pub type JournalEntryOf<T> =
+		JournalEntry<JournalDetailsOf<T>, EntryHashOf<T>, BlockNumberOf<T>>;
+	pub type ScoreEntryOf = ScoreEntry<CountOf, ScoreOf>;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
```
Comment (smohan-dw) [inline pallets/score/src/lib.rs:76]: We have to change the account details to use DIDs

Comment (rohibhan) [reply pallets/score/src/lib.rs:76]: Thanks. Still working on it.

**Comment by NiranjanAP** [comment]
This PR can be cancelled I think? you have the changes in the other PR right?

**Comment by rohibhan** [comment]
PR is not required anymore as the changes have been moved to another PR

**Pull request closed without merging**